package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "lerna-test": "lerna run setup-test && lerna run test",
     "lerna-bootstrap": "lerna bootstrap",
-    "lint": "find . -type f -name \"*.js\" -not -path \"*node_modules*\" | xargs eslint"
+    "lint": "find . -type f -name \"*.js\" -not -path \"*node_modules*\" | xargs eslint --fix"
```

Comment:
Might be pointless, since the auto-fixed changes won't be committed, yet the build will pass. I think it's better to let people fix their style problems by themselves.

Reply:
Yeah I agree, although we could add  a separate `lint-fix` task

Reply:
Thanks for pointing out the auto-fixed changes not being committed, I had forgotten that part.

So I think the least cumbersome way of doing it is creating another script can be:

```json
"lint-fix": "find . -type f -name \"*.js\" -not -path \"*node_modules*\" | xargs eslint --fix"
```

Where the contributor can run it to make sure the code added complies with the repository style.

also instead of disabling the `max-len` rule I updated it and added

```js
"max-len": [
  "error",
    {
      "code": 120
    }
],
```

That way there's a limit that is not disruptive to the current codebase.

https://github.com/firebase/snippets-node/blob/9ae3a00985b53025fdc82716046882af71b6009d/firestore/solution-scheduled-backups/app.js#L47

contains a line that's >= 110, and other files.

Reply:
Actually found a way to commit auto-fixed changes. We can use husky and lint-staged to run `lint-fix` before committing. @IvanBila please have a look at IvanBila/snippets-node#1